runtime/caml/blake2.h

```diff
@@ -0,0 +1,46 @@
+/**************************************************************************/
+/*                                                                        */
+/*                                 OCaml                                  */
+/*                                                                        */
+/*           Xavier Leroy, Collège de France and Inria Paris              */
+/*                                                                        */
+/*   Copyright 2022 Institut National de Recherche en Informatique et     */
+/*     en Automatique.                                                    */
+/*                                                                        */
+/*   All rights reserved.  This file is distributed under the terms of    */
+/*   the GNU Lesser General Public License version 2.1, with the          */
+/*   special exception on linking described in the file LICENSE.          */
+/*                                                                        */
+/**************************************************************************/
+
+/* BLAKE2 message digest */
+
+#ifndef CAML_BLAKE2_H
+#define CAML_BLAKE2_H
+
+#ifdef CAML_INTERNALS
+
+#include "mlvalues.h"
```

Comment:
Nit: 
```suggestion
#include "misc.h"
```
(just `CAMLextern` is needed)

Reply:
Makes sense.  Done in d2a89ee845